pkg/convert/pprof/ctx.go

```diff
@@ -0,0 +1,16 @@
+package pprof
+
+import "context"
+
+type ctxPprofLineNumbersKey string
+
+const ctxPprofLineNumbers ctxPprofLineNumbersKey = "ctxPprofLineNumbers"
+
+func WithLineNumbersEnabled(ctx context.Context, v bool) context.Context {
+	return context.WithValue(ctx, ctxPprofLineNumbers, v)
+}
+
+func LineNumbersEnabledFromContext(ctx context.Context) bool {
+	v, ok := ctx.Value(ctxPprofLineNumbers).(bool)
+	return ok && v
+}
```

Comment:
Passing configuration parameters via context might not be desirable:
 - https://www.ardanlabs.com/blog/2019/09/context-package-semantics-in-go.html.
 - https://dave.cheney.net/2017/01/26/context-is-for-cancelation

Please consider the following implementation:

```go
type Option func(*options)

type options struct {
  includeLineNumber bool
}

func WithLineNumber() func(*options) {
   return func(o *options) {
      o.includeLineNumber = true
   }
}

func (p *Parser) Convert(ctx context.Context, startTime, endTime time.Time, profile *tree.Profile, options ...Option) error
```

Reply:
I removed this option for non rbspy